doc/cli.markdown

```diff
@@ -1406,6 +1406,13 @@ Disables check for matching architecture in image and application
 
 Pin the preloaded device (not application) to the preloaded release on provision
 
+#### --add-certificate &#60;certificate.crt&#62;
+
+Add the given certificate (in PEM format) to /etc/ssl/certs in the preloading container.
+The file name must end with '.crt' and must not be already contained in the preloader's
+/etc/ssl/certs folder.
+Can be repeated to add multiple certificates.
```

Comment:
Apparently the last PR (#1262) also hadn't run `npm build` to update `cli.markdown`.